app/metrics.yaml

```diff
@@ -841,6 +841,17 @@ sync_auth:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2020-09-01"
+  use_email:
+    type: event
+    description: >
+      A user chose to use their email to sign in instead of scanning a QR code
```

Comment:
Maybe mention the pairing counter-part of this event? I'm guessing it's `paired`, which is a little confusing - that name implies that `paired` is logged after user successfully signed-in via pairing. This event, I'm assuming, is emitted when user begins to login via email. So, they're not 1-1 comparable? Or is it just bad naming?

Reply:
The counter part is `scan_pairing`.

`paired` counterparts are `sign_in` , `sign_up` or external, etc